pkg/cli/migrate.go

```diff
@@ -0,0 +1,33 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/replicatedhq/embedded-cluster-operator/pkg/migrations"
+	"github.com/spf13/cobra"
+)
+
+func MigrateCmd() *cobra.Command {
+	var migration string
+
+	cmd := &cobra.Command{
+		Use:          "migrate",
+		Short:        "Run the specified migration",
+		SilenceUsage: true,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			err := migrations.Run(cmd.Context(), migration)
+			if err != nil {
+				return fmt.Errorf("migration %q failed: %w", migration, err)
+			}
+			return nil
+		},
+	}
+
+	cmd.Flags().StringVar(&migration, "migration", "", "The migration to run")
```

Comment:
Would be better to have subcommands rather than a string to determine which migration to run. This is how we did it in KOTS: https://github.com/replicatedhq/kots/blob/fcfe61927e4bafe8da21412505f620c7e4992183/cmd/kotsadm/cli/migrate.go#L13-L27